### ISSUE TITLE
Log output of worker to file

### ISSUE THREAD

**Issue by cogitantium** [d502e19/scraper#52]
The output from a worker, such as:
```
Worker W1 has started
Worker W1 received task https://old.reddit.com/
Worker W1 received task https://old.reddit.com/#content
Worker W1 received task https://old.reddit.com/subreddits/
Worker W1 received task https://old.reddit.com/r/popular/
Worker W1 received task https://old.reddit.com/r/all/
Worker W1 received task https://old.reddit.com/r/random/
Worker W1 received task https://old.reddit.com/users/
Worker W1 received task https://old.reddit.com/r/AskReddit/
Worker W1 received task https://old.reddit.com/r/news/
Worker W1 received task https://old.reddit.com/r/pics/
Worker W1 received task https://old.reddit.com/r/funny/
```

Might be useful for debugging once containerised. Should be fairly simple to implement logging this output to a file, e.g.; `worker{num}.log`.

**Comment by cogitantium** [comment]
Should also allow for different log-levels, such as; error, warn, info, debug, and trace, which is already implemented by Display for ScraperErrors.

**Comment by cogitantium** [comment]
Related to #33

**Comment by cogitantium** [comment]
Was closed in #62
